src/client/components/annotations.ts

```diff
@@ -100,6 +102,9 @@ export class Annotations extends LitElement {
   @property({ type: Object, reflect: true })
   validationErrors?: CellAnnotationsErrorResult
 
+  @property({ type: Number })
+  cellIndex?: number
```

Comment:
We cannot use `cellIndex` because it will change as we add more cells to the document.

It would be preferable to use `uuid` instead for this reason.

Reply:
Good catch, fixed